---
title: The landscape of gene expression variation in human populations
bibliography: [./references.bib]
csl: [./evolution.csl]
header-includes:
- \usepackage[backref=true]{biblatex}
- \DefineBibliographyStrings{english}{ backrefpage = {page}, backrefpages = {pages}}
mainfont: Skolar PE TEST Regular
sansfont: Skolar Sans PE TEST
mainfontoptions:
- Numbers=Lowercase
- Numbers=Proportional
geometry:
- top=20mm
- left=25mm
- right=25mm
- bottom=20mm
link-citations: true
---

# Intro

Molecular phenotypes such as gene expression are a powerful tool for understanding physiology, disease, and evolutionary adaptations. In this context, average trait values are usually the focus of investigation, while variation is treated as a nuisance. However, gene expression variation can be directly involved in determining fitness [@Wang2011-ts], and the relation between gene expression variation can be a powerful indicator of gene environment interactions and disease[@Lea2019-pq]. Understanding the landscape of gene expression variation, and how variable it is across genes and across human populations is then a neglected avenue to understand biological evolution and our relation to the environment.

While mean gene expression levels are the main target of analysis, differences in the variation of gene expression can also be  relevant to fitness [@Wang2011-ts]. 

Structure of genetic variation can affect phenotypes (and fitness). Gene are expected to differ in their gene expression variance due to several factors: genetic variation, selection, GxE interactions, robustness of gene expression regulation... Can we expect that the pattern of genetic variation across populations to be similar? Which genes are expected to change more or less? Which genes are expected to be more or less variable?

Here we use public gene expression data sets to evaluate how the differences in gene expression variation is structures across several independent studies. We collected and compared the gene expression variation across several studies, and used the similarities across these studies to create a gene expression variation ranking, which orders genes from least variable to most variable. We then explore the expected drivers of this gene expression ranking, showing that both cis and trans regulation are involved with the determination of gene expression variance. 


# Methods

## Data sources

We selected 60 studies with large sample sizes from public gene expression repositories recount3 [@Wilks2021-uj] and Expression Atlas [@Papatheodorou2020-dn].

We use studies to refer to independent data sets, which could have been generated by the same consortium. For example, the gTEX data are separated by tissue, and we refer to each tissue as a separate study. The same applies for TCGA data. 

## Data processing pipeline

We use a standardized pipeline to measure gene expression variation while removing extraneous sources of variation.

Case-control studies were filtered to keep only control samples. 

Filtering by min count per million (cpm) reads and mean cpm. Variance stabilizing transformation from DESeq2 [@Love2014-mp]. Fixed effects correction. Outlier removal using [@Chen2020-fy]. Gene expression variance is measured in the residuals after fixed effect correction and outlier removal.

## Gene connectivity

As a proxy for the degree of trans regulation that each gene is subjected to, we calculate the average weighted connectivity for all genes. To do this, for each study, we create a fully connected gene-by-gene graph in which each edge is weighted by the Spearman correlation between gene expression. We then trim this graph by keeping only edges for which the Spearman correlation is significant at a false discovery rate of 1%. In this trimmed network, we then take the average of the Spearman correlation of all remaining edges for each gene. So, for each study we have a measure of the average correlation of each gene with every other gene. The average connectivity for each gene is the average across all studies in which that gene is expressed.

## Variance correlation

We assessed the similarity in gene expression variation across studies by using a between study Spearman correlation matrix. using Spearman correlations avoid problems related to overall scaling or coverage differences, and allows us to assess if the same genes are usually more or less variable across studies.

PCoA of studies using the Spearman correlation matrix.

To investigate the factors involved in determining correlations between studies, we used a varying effects model to investigate the effect of study origin and tissue on the correlations across studies.

## Gene Ontology enrichment

# Results

![Density plot of standard deviations after z-normalization. Inset plot shows distribution of mean centered standard deviations grouped by study without normalization. The corresponding rug plots show the location of the highest ranking gene in standard deviation rank (blue) and lowest (red).](figures/sd_dist.png){ width=50% }

## Correlation between variation across studies

![Correlation plot showing the cross study Spearman rank correlation of standard deviations after filtering and batch correction](figures/corr_plot.png){ width=50% }

![Standard deviation correlation PCoA](figures/sd_PCoA_plot.png){ width=50% }

# GO enrichment

![Gene set enrichment analyses testing for over representation of gene ontology categories in the upper and lower 5% quantiles of the gene variation rank](figures/local_go_lowerUpper.png){ width=50% }

## Gene level statistics

PopHuman stuff: pi and ... [@Casillas2018-nk]

![Violin plot showing the relationship between SD rank and mean Ï€ value for genes](figures/pi.png){ width=50% }

\newpage

# Discussion

Gene expression variance is reasonably conserved across studies.
Gene expression variance is predictive of biological function.
Gene expression variance can be partially explained by genetic variation and genetic associations between gene expression.


# References





